bin/interop-decode.c

```diff
@@ -365,12 +366,9 @@ main (int argc, char **argv)
                 free(buf);
                 break;
             case LQRHS_BLOCKED:
-                buf->off += (unsigned) (p - buf->buf);
```

Comment:
This is a regression: when "blocked" is returned, you're not supposed to call read() again.

Reply:
I will decouple the cases so that `case LQRHS_BLOCKED` breaks and `case LQRHS_NEED` loops.